pinot-core/src/main/java/org/apache/pinot/core/data/table/IndexedTable.java

```diff
@@ -33,43 +33,41 @@
  */
 public abstract class IndexedTable implements Table {
 
-  List<AggregationFunction> _aggregationFunctions;
+  AggregationFunction[] _aggregationFunctions;
```

Comment:
kept them as package-private because they're not and will not be used in any other packages.
Also, the IDE flags them if they are made protected as "Access can be package-private"